pages/index.tsx

```diff
@@ -1,33 +1,29 @@
 import type { NextPage } from "next";
 import Form from "../components/Form";
-// @ts-ignore
-import MarkdownField from "../components/MarkdownField";
-import useStore from "../zustand/store";
-import styled from "styled-components";
+import Typography from "@mui/material/Typography";
+import Grid from "@mui/material/Grid";
+import MarkDownOutput from "../components/MarkdownOutput";
+import WikiUpdater from "../components/WikiUpdater";
 
 const Home: NextPage = () => {
   return (
-    <>
-      <StyledMain>
-        <Styledh1>Github Wiki Generator</Styledh1>
+    <Grid container p={6}>
```

Comment:
Is this grid thingy something to faster create Grid Boxes with MUI?

Reply:
It is the component MUI offers for Grid based Layouts, there is also Stack for flexbox Layouts. the cool thing about it are the easy to set breakpoints:

https://mui.com/material-ui/customization/breakpoints/

Which I also utilize. xs={12} means 100% of the available width on lowest screen size, xs={6} means half of the available space. 12 ist the maximum, but we can chat about it in person if you want to,.